detail_views/views.py

```diff
@@ -14,16 +14,21 @@
 # Create your views here.
 from detail_views.forms import *
 
-#token = {'Authorization': 'Token eadb51c79b15f8eb55d49e0a9228beea6b468c64'}
 
 
 def model_detail_page(request, pk=None):
     header2 = {'Authorization': request.COOKIES['token']}
 
-    modeldata = requests.get('http://' + request.get_host() + '/models/', headers=header2, params={'pk': pk})
+    print('pk: '+pk)
+
+    # pk = int(pk)
+    # print(pk)
+
+    modeldata = requests.get('http://' + request.get_host() + '/models/'+pk+'/', headers=header2)
     instrumentsdata = requests.get('http://' + request.get_host() + '/instruments/', headers=header2, params={})
```

Comment:
Why are you also getting the instruments data? The model data contains an `instruments` field. This is a redundant query.